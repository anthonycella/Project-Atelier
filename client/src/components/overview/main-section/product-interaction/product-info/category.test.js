import renderer from 'react-test-renderer';
import Category from './category.jsx';

it('Category renders to the page', () => {
  const component = renderer.create(
    <Category />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
