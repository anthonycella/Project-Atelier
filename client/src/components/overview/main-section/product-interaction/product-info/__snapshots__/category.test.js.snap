// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category renders to the page 1`] = `
<div>
  CATEGORY
</div>
`;
